wagtail/wagtailforms/views.py

```diff
@@ -56,7 +56,7 @@ def list_submissions(request, page_id):
         for field in form_page.form_fields.all()
     ]
 
-    submissions = FormSubmission.objects.filter(page=form_page)
+    submissions = FormSubmission.objects.filter(page=form_page).order_by('submit_time')
```

Comment:
I think it would probably make sense to have most recent submissions on the first page